lib/src/constants/sheba/constants.dart

```diff
@@ -0,0 +1,319 @@
+var _banksInfo = <Map<String, dynamic>>[
+  {
+    'nickname': 'central-bank',
+    'name': 'Central Bank of Iran',
+    'persianName': 'بانک مرکزی جمهوری اسلامی ایران',
+    'code': '010',
+    'accountNumberAvailable': false
+  },
```

Comment:
I think we talked about this scenario. and I think it's better to use model classes instead of `Map<String, dynamic>`.
@hadi-norouzi  what do u think about it?

Reply:
I think because the keys are long while the user reads from the `Map`s, it is better to use model classes instead.
using `.` operator (member access operator) is easier than writing long keys, I will check it soon.

Reply:
I'm agree with model classes